omniNotes/src/androidTest/java/it/feio/android/omninotes/Screen/Home.java

```diff
@@ -0,0 +1,43 @@
+package it.feio.android.omninotes.Screen;
+
+import it.feio.android.omninotes.Helpers.Constants;
+import it.feio.android.omninotes.Helpers.HelpersMethods;
+import it.feio.android.omninotes.R;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withContentDescription;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+
+public class Home {
+    public static boolean isMenuSortInTheRightOfSearch() {
+        return HelpersMethods.isFirstMatcherToTheRightOfSecondMatcher(withId(R.id.menu_sort), withId(R.id.menu_search));
+    }
+
+    public static boolean isMenuSortInTheLeftOfMoreOptions() {
+        return HelpersMethods.isFirstMatcherToTheLeftOfSecondMatcher(withId(R.id.menu_sort), withContentDescription(Constants.MORE_OPTIONS));
+    }
+
+    public static void clickSearchButtonAndSearch() {
```

Comment:
This method only clicks the search button. It doesn't search for items

Reply:
I changed.